index.html

```diff
@@ -1,697 +1,732 @@
-            <small>Matthew Beedham - 22 minutes ago</small>
+        <div class="buttons">
+            <div class="search">
+                <button type="button"><img src="images/search.png" alt=""></button>
+            </div>
+            <div class="menu-hamburguer">
+                <button type="button"><img src="images/menu-hamburguer.png" alt=""></button>
+            </div>
         </div>
 
-        <div class = "n1">
-            <img src = "images/news-3.png" alt = "">
-            <h4>Uber expands its low-emission ride options to 1,400 more North American cities</h4>
-            <small>Cities Today - 5 hours ago</small>
+        <div class="bottom">
+            <p>Calling young digital talent in the Netherlands - Be recognized in the T500 ￫</p>
         </div>
 
-        <div class = "n1">
-            <img src = "images/news-4.png" alt = "">
-            <h4>3 reasons why almost every VC investor passed on Zoom</h4>
-            <small>Philip Levinson -  5 hours ago</small>
-        </div>
+    </nav>
 
-        <div class = "n1">
-            <img src = "images/news-5.png" alt = "">
-            <h4>It's happening! World's biggest diesel factory pivots to electric motors</h4>
-            <small>Matthew Beedham -  6 hours ago</small>
-        </div>
+    <!--Highlights-->
 
-        <div class = "n1">
-            <img src = "images/news-6.png" alt = "">
-            <h4>This nearly inexplicable exoplanet has the density of cotton candy</h4>
-            <small>The Cosmic Companion - 6 hours ago</small>
-        </div>
+    <div class="highlights">
 
-        <div class = "n1">
-            <img src = "images/news-7.png" alt = "">
-            <h4>The future of US goverment vehicles is electric</h4>
-            <small>Matthew Beedham - 7 hours ago</small>
+        <div class="article-1">
+            <img class="show-when-big" src="images/img-1.png" alt="">
+            <img class="show-when-small" src="images/art-1-nored.png" alt="logo">
         </div>
-
-        <div class = "n1">
-            <img src = "images/news-8.png" alt = "">
-            <h4>Signal is drama-free for now, but it should prepare for the worst</h4>
-            <small>Abhimanyu Ghoshal - 8 hours ago</small>
+        <div class="article2-3">
+            <img src="images/img-2.png" alt="">
+            <img src="images/img-3.png" alt="">
         </div>
 
-     </div>
-
-
-
-     <!--Latest Funding-->
-
-     <div class="subtitle-links">
-        <h2><a href ="">Latest funding rounds on Index.com ></a></h2>
-            <ul class = "latest">
-            <li><a href="">Add info</a> </li>|
-            <li><a href="">Get updates by email</a></li>|
-            <li><a href="">See more</a></li>
-        </ul>
-        </div>   
-
+    </div>
 
-     <div class="latest-funding">
+    <main class="main-content">
+        <!--Latest News-->
+        <div class="news-content">
+            <h2 class="subtitle-links"><a href="">Latest news ></a> </h2>
```

Comment:
#### In mobile resolution latest news section
- [ ] Each article should take the full width and are aligned in a column
- original 👇 
- ![image](https://user-images.githubusercontent.com/64360974/114287728-db876e80-9a71-11eb-9a50-771d0cc2251a.png)
- Yours 👇 
- ![image](https://user-images.githubusercontent.com/64360974/114287776-62d4e200-9a72-11eb-88f7-7ffaac5b36af.png)